src/main/java/org/csanchez/jenkins/plugins/kubernetes/AllContainersRunningPodWatcher.java

```diff
@@ -68,55 +66,7 @@ private void updateState(Pod pod) {
     }
 
     boolean areAllContainersRunning(Pod pod) {
-        PodStatus podStatus = pod.getStatus();
-        if (podStatus == null) {
-            return false;
-        }
-        List<ContainerStatus> containerStatuses = pod.getStatus().getContainerStatuses();
-        if (containerStatuses.isEmpty()) {
-            return false;
-        }
-        for (ContainerStatus containerStatus : containerStatuses) {
-            if (containerStatus != null) {
-                ContainerStateWaiting waitingState = containerStatus.getState().getWaiting();
-                if (waitingState != null) {
-                    String waitingStateMsg = waitingState.getMessage();
-                    if (waitingStateMsg != null && waitingStateMsg.contains("Back-off pulling image")) {
-                        String errorMsg = "Unable to pull Docker image \""+containerStatus.getImage()+"\". Check if image tag name is spelled correctly.";
-                        runListener.error(errorMsg);
-                        throw new InvalidDockerImageException(errorMsg);
-                    }
-                    return false;
-                }
-                if (containerStatus.getState().getTerminated() != null) {
-                    return false;
-                }
-                if (!containerStatus.getReady()) {
-                    return false;
-                }
-            }
-        }
-        return true;
-    }
```

Comment:
Moved to https://github.com/jenkinsci/kubernetes-plugin/pull/772/files#diff-55ff61b56516ad2fa5dd36df80c3836cR243